# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vidix/gcc4.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- vidix-0.9.9.1/vidix/drivers/pm3_vid.c.orig	2004-01-10 19:17:33.000000000 +0100
+++ vidix-0.9.9.1/vidix/drivers/pm3_vid.c	2005-09-05 20:00:22.000000000 +0200
@@ -50,7 +50,7 @@
 
 static pciinfo_t pci_info;
 
-static void *pm3_reg_base;
+/*static*/ void *pm3_reg_base;
 static void *pm3_mem;
 
 static int pm3_vidmem = PM3_VIDMEM;
